packages/modules/src/modules/epics/ContributionsEpicChoiceCards.tsx

```diff
@@ -8,6 +8,7 @@ import {
 import { getLocalCurrencySymbol } from '@sdc/shared/dist/lib/geolocation';
 import { css } from '@emotion/react';
 import { until } from '@guardian/src-foundations/mq';
+import { countryCodeToCountryGroupId } from '@sdc/shared/dist/lib';
```

Comment:
I'd do a search and replace on `shared/dist` to replace with just `shared`

Reply:
✅